### DISCUSSION TITLE
[Feature Request]:  转移和辅种通知增加仅失败

### DISCUSSION THREAD

**Discussion by paysenyu** [jxxghp/MoviePilot#3366] [讨论]
### 当前程序版本

1.18

### 功能改进

建议参考iyuu的通知, 转移和辅种加上仅失败时候发送通知, 成功默认不通知.

**Comment by dlovew** [comment]
iyuu通知是可选的并不是成功默认不通知

**Comment by paysenyu** [comment]
> iyuu通知是可选的并不是成功默认不通知

是的，所以希望增加一个
